samples/matter/weather_station/weather-station.zap

```diff
@@ -18,13 +18,13 @@
   "package": [
```

Comment:
Can this file be moved to `src/`? That way the `generate.py` run on this file will generate the code in the correct location.

Reply:
Sure, done.